sway/commands.c

```diff
@@ -278,10 +279,17 @@ list_t *execute_command(char *_exec, struct sway_seat *seat,
 			argv[i] = do_var_replacement(argv[i]);
 		}
 
-		if (!config->handler_context.using_criteria) {
+
+		if (!using_criteria) {
 			// The container or workspace which this command will run on.
-			struct sway_node *node = con ? &con->node :
-					seat_get_focus_inactive(seat, &root->node);
+			struct sway_node *node;
+			if(con) {
```

Comment:
Nit: space after the `if`.

Nit: no need to define `struct sway_node *node`, can just call `set_config_node` directly in both branches.